modules/nf-core/atlas/splitmerge/meta.yml

```diff
@@ -7,58 +7,71 @@ keywords:
   - read group
 tools:
   - "atlas":
-      description: "ATLAS, a suite of methods to accurately genotype and estimate genetic diversity"
+      description: "ATLAS, a suite of methods to accurately genotype and estimate genetic
+        diversity"
       homepage: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
       documentation: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
       tool_dev_url: "https://bitbucket.org/wegmannlab/atlas"
       doi: "10.1101/105346"
-      licence: "['GPL v3']"
+      licence: ["GPL v3"]
+      identifier: biotools:atlas_db
 input:
-  - meta:
-      type: map
-      description: |
-        Groovy Map containing sample information
-        e.g. [ id:'test', single_end:false ]
-  - bam:
-      type: file
-      description: Single input BAM file.
-      pattern: "*.bam"
-  - bai:
-      type: file
-      description: The BAI file for the input BAM file
-      pattern: "*.bai"
-  - read_group_setting:
-      type: file
-      description: |
-        TXT file containing the split and merge settings for
-        each readgroup. Each line consist of one readgroup,
-        single/double identifier and the maximum cycle number
-        of the sequencer. e.g. "RG1 single 100"
-      pattern: "*.txt"
-  - blacklist:
-      type: file
-      description: |
-        blacklist.txt (optional), A txt file with blacklisted read names
-        that should be ignored and just written to file, each on a new line
-      pattern: "*.txt"
+  - - meta:
+        qualifier: val
+        type: map
```

Comment:
@bentsherman - does this look about right to you, in terms of what we call things here?

Reply:
what's the use of adding the qualifier when you already have the type?

Reply:
The qualifier can be useful to know if two modules have the same inputs and outputs. 
For example, we could have the case where two different outputs have type string, but the qualifier is val and env.

Reply:
You mean to check if the modules can be chained? In that case it doesn't matter if it a `val` or `env`, as long as they are both strings

Reply:
I am more thinking of a setting where modules can be interchangeable, for example for benchmarking of different tools. I would like to make sure that using aligner1 and using aligner2, both inputs and outputs will be exactly the same.
I know this can already be done with if/else and operating the channels if needed. But in the future, it would be cool if we can use this to automatically add any module which matches the requirements.

Reply:
Even there it doesn't matter if the string input is a `val` or `env`, as long as it is a string they are interchangeable.

The only difference is that val says "provide this value as a variable" whereas env says "provide this value as an environment variable", but those details are internal to the process

Reply:
OK, I see what you mean! Would it be best to remove the `qualifier`?

Reply:
yes I think so